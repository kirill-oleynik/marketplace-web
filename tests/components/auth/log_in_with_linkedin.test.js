const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;

const {
  LogInWithLinkedin
} = require('../../../app/components/auth/log_in_with_linkedin');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <LogInWithLinkedin
        t={(translation) => translation}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
