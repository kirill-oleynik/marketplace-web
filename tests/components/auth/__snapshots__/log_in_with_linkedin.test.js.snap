// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<a
  className="button-login-with-socials button-login-with-socials--linkedin"
  href="undefined/auth/linkedin"
>
  <i
    className="icon icon-linkedin button-login-with-socials__icon"
  />
  <span>
    linkedin
  </span>
</a>
`;
